sdk/automation/Azure.ResourceManager.Automation/tests/AutomationManagementTestBase.cs

```diff
@@ -19,11 +19,13 @@ public class AutomationManagementTestBase : ManagementRecordedTestBase<Automatio
         protected AutomationManagementTestBase(bool isAsync, RecordedTestMode mode)
         : base(isAsync, mode)
         {
+            JsonPathSanitizers.Add("$..uri");
```

Comment:
Why does Uri need to be sanitized?

Reply:
This uri contains token info, and the pipeline will throw `Credential Scanner Error CSCAN-AZURE0130`